test/integration/targets/docker_swarm/aliases

```diff
@@ -3,3 +3,5 @@ skip/osx
 skip/freebsd
 destructive
 skip/rhel8.0
+skip/docker
```

Comment:
Append a comment to the end of this line to explain why the tests are skipped.

Reply:
Done. I didn't knew one can put comments into the `aliases` file...